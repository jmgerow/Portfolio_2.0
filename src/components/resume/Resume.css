.resume {
    margin-bottom: 200px;
    width: 1000px;
    height: 2300px;
}