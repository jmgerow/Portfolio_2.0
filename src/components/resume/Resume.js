import React, { Component } from 'react';

import resume from '../../images/Justin_M_Gerow_-_Resume_v2_web.pdf';

import './Resume.css';

class Resume extends Component {

    render() {
        return (
            <div className="container">
                <div className="row blue-grey-text text-darken-1" id="title">
                    <h3>Resume</h3>
                </div>
                <div className="row">
                    <div className="col s12 center">
                        <embed src={resume} class="resume" />
                    </div>
                </div>
            </div>
        );
    };
};

export default Resume;