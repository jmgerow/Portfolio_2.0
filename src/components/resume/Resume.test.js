import React from 'react';
import { shallow } from 'enzyme';
import Resume from './Resume';

describe('Resume', () => {
    const resume = shallow(<Resume />)
  
    it('renders without crashing', () => {
      expect(resume).toMatchSnapshot();
    });
 
  });