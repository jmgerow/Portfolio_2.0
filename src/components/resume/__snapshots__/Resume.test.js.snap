// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Resume renders without crashing 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Resume />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="row blue-grey-text text-darken-1"
          id="title"
        >
          <h3>
            Resume
          </h3>
        </div>,
        <div
          className="row"
        >
          <div
            className="col s12 center"
          >
            <embed
              class="resume"
              src="Justin_M_Gerow_-_Resume_v2_web.pdf"
            />
          </div>
        </div>,
      ],
      "className": "container",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <h3>
            Resume
          </h3>,
          "className": "row blue-grey-text text-darken-1",
          "id": "title",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Resume",
          },
          "ref": null,
          "rendered": "Resume",
          "type": "h3",
        },
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="col s12 center"
          >
            <embed
              class="resume"
              src="Justin_M_Gerow_-_Resume_v2_web.pdf"
            />
          </div>,
          "className": "row",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <embed
              class="resume"
              src="Justin_M_Gerow_-_Resume_v2_web.pdf"
            />,
            "className": "col s12 center",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "class": "resume",
              "src": "Justin_M_Gerow_-_Resume_v2_web.pdf",
            },
            "ref": null,
            "rendered": null,
            "type": "embed",
          },
          "type": "div",
        },
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="row blue-grey-text text-darken-1"
            id="title"
          >
            <h3>
              Resume
            </h3>
          </div>,
          <div
            className="row"
          >
            <div
              className="col s12 center"
            >
              <embed
                class="resume"
                src="Justin_M_Gerow_-_Resume_v2_web.pdf"
              />
            </div>
          </div>,
        ],
        "className": "container",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <h3>
              Resume
            </h3>,
            "className": "row blue-grey-text text-darken-1",
            "id": "title",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Resume",
            },
            "ref": null,
            "rendered": "Resume",
            "type": "h3",
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <div
              className="col s12 center"
            >
              <embed
                class="resume"
                src="Justin_M_Gerow_-_Resume_v2_web.pdf"
              />
            </div>,
            "className": "row",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <embed
                class="resume"
                src="Justin_M_Gerow_-_Resume_v2_web.pdf"
              />,
              "className": "col s12 center",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "class": "resume",
                "src": "Justin_M_Gerow_-_Resume_v2_web.pdf",
              },
              "ref": null,
              "rendered": null,
              "type": "embed",
            },
            "type": "div",
          },
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
